tests/bsim/bluetooth/mesh/src/mesh_test.h

```diff
@@ -202,6 +204,12 @@ void bt_mesh_test_ra_cb_setup(void (*cb)(uint8_t *, size_t));
 
 uint16_t bt_mesh_test_own_addr_get(uint16_t start_addr);
 
+void bt_mesh_test_send_over_adv(void *data, size_t len);
+/* Wait for an adv sent by another device (i. e. using `bt_mesh_test_send_over_adv`).
```

Comment:
Description is not relevant. It is not for receiving advertising only but for receiving frames over gatt too.

Reply:
Updated the description and naming. Better now?